src/features/landing/components/AboutUs/Info/Info.tsx

```diff
@@ -0,0 +1,57 @@
+'use client';
```

Comment:
did you add this only for useTranslation hook? hmm, this can be a problem with translation, try research this question, will useTranslation works on server or we need another approach for localization

Reply:
In progress.